components/infobox/extensions/commons/infobox_extensions_prize_pool.lua

```diff
@@ -74,9 +77,7 @@ function PrizePoolCurrency.display(args)
 		Variables.varDefine('tournament_currency_date', date)
 		Variables.varDefine('tournament_currency_text', text)
 		Variables.varDefine('tournament_prizepoollocal', prizepool or '')
-
-		local prizepoolUsdValue = string.gsub(prizepoolUsd or '', ',', '')
```

Comment:
Removing this will not break manual inputs?

Reply:
the `_cleanValue` calls on the manual input at the top already solve it